tensorflow/core/kernels/reduction_gpu_kernels.cu.h

```diff
@@ -13,17 +13,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#if GOOGLE_CUDA
+#if defined GOOGLE_CUDA || defined TENSORFLOW_USE_ROCM
```

Comment:
how about `#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM`? easier for future global search-and-replace